0n0-CART.Rmd

```diff
@@ -0,0 +1,263 @@
+# CART
+```{r}
+library(ggplot2)
+library(tree)
+```
+
+```{r}
+# Data
+summary(iris)
+ggplot(data = iris) + geom_point(aes(x = Petal.Length, y = Petal.Width, col = Species))
+```
+
+The idea behind CART is to divide the predictor space (petal width and length) with a straight line and fit simple models on either side of the line. 
+
+Mathematically, consider predictors $X_j$ and some split point $s$ that splits the predictor space into two half-spaces $$L_{j, s} = \lbrace X |X_j \le s \rbrace \text{ and } R_{j, s} = \lbrace X |X_j > s\rbrace$$. 
+
+The idea is to split on the varible and at the location which minimizes some loss function: $$min_{j, s}\lbrace loss(R_{j, s}) + loss(L_{j, s}) \rbrace$$
+For classification problems, the easiest model is ``majority wins'' and the loss function is the number of misclassificed observations. For regression, the easiest model is the mean model, and the loss function is squared error loss.
+
+
+```{r}
+# Grow a tree
+classTree = tree(Species ~ Sepal.Length + Sepal.Width + Petal.Length + Petal.Width, data = iris)
+summary(classTree)
```

Comment:
separate summary, plot, and text into separate code chunks.
this way the output is right below the code